debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -273,6 +274,17 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withDescription("The timezone of the server used to correctly shift the commit transaction timestamp on the client side"
                     + "Options include: Any valid Java ZoneId");
 
+    public static final Field SKIP_LSN_TIMESTAMP_QUERY = Field.create(DATABASE_CONFIG_PREFIX + SKIP_LSN_TIMESTAMP_QUERY_CONFIG_NAME)
+            .withDisplayName("Skip LSN timestamp query")
+            .withDefault(Boolean.FALSE)
+            .withType(Type.BOOLEAN)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withDescription("Skips the query which maps a commit LSN to a point in time when the commit happened." +
+                    "Options include:" +
+                    "'true', which means the LSN timestamp is defaulted to Instant.now()." +
```

Comment:
Should be updated as per above. Also let's avoid exposing internals like the usage of `Instant`: "defaulted to the timestamp of processing this change event". Non-Java users wouldn't know what it is anyways.

Reply:
Nice suggestion. Thanks!